python/raydp/torch/estimator.py

```diff
@@ -119,6 +122,14 @@ def __init__(self,
                Note: Now the value can only be False
         :param num_processes_for_data_loader: the number of processes use to speed up data loading
         :param callbacks: which will be executed during training.
+        :param metrics_name: the name of metrics' classes used to evaluate model. the name can be
+               used refer: https://torchmetrics.readthedocs.io/en/latest/, For example:
```

Comment:
What about changing "the name can be used refer:" to "The full name list is available at:".

Reply:
okay